tests/UI/pages/commonPage.js

```diff
@@ -243,11 +243,12 @@ module.exports = class CommonPage {
   /**
    * Go to Page and wait for navigation
    * @param selector
+   * @param waitUntil
    * @return {Promise<void>}
    */
-  async clickAndWaitForNavigation(selector) {
+  async clickAndWaitForNavigation(selector, waitUntil = 'load') {
```

Comment:
It's the default setting.

Reply:
Yes, but it aims to be optional in the method content

Reply:
yes, it's default setting for [waitUntil](https://playwright.dev/#version=v1.1.1&path=docs%2Fapi.md&q=pagewaitfornavigationoptions) , and optional here